### PR TITLE
Added native pipe where compatible

### PR DESCRIPTION
**Pull request by SimonAytes**
### Overview

What problem or feature request does this pull request resolve?

### Issues Addressed

 * #133 

### Changes Addressed/Proposed

 * 

### Reproducible Examples

Demonstrate how the new code performs.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

### Tests

Include test cases, and expected output

### CODE REVIEW

File: R/fstadv.R [hunk 1]
```diff
@@ -237,17 +237,17 @@ fstadv_forecasts <- function(content, key, adv, adv_date) {
...
   # storm is expected to degenerate or already has).
   forecasts <-
-    content %>%
-    stringr::str_match_all(pattern = ptn) %>%
+    content |>
+    stringr::str_match_all(pattern = ptn) |>
     # # Get only the columns needed excluding the matched string
-    # purrr::map(`[`, , 2:22) %>%
+    # purrr::map(`[`, , 2:22) |>
     # If any storm has 0 forecasts (i.e., the list element is empty), populate
```
Comment (elinw) [inline R/fstadv.R:244]: I think you nee to stop the pipe chain at 241, then do the 243 stuff, then restart at 243.

Comment (SimonAytes) [reply R/fstadv.R:244]: It looks like line 243 is commented out, so it won't be compiled.

Should 241 pipe in to 246?

Comment (elinw) [reply R/fstadv.R:244]: I went through the code and found a number of places that were subsetting using `[]` so in the end I don't think it is just here ... but then again whatever you did seemed to work.  Take a look at my other branch called pipe2 #173

**Review by elinw — APPROVED** [review]
It's a lot, the build will probably fail but we'll all have a fresh base,

**Pull request merged**